index.bs

```diff
@@ -4401,6 +4401,10 @@ for {{DOMException}}, a description, and legacy code values.
 
 Note: If an error name is not listed here, please file a bug as indicated at the top of this specification and it will be addressed shortly. Thanks!
 
+Note: Do not confuse {{SyntaxError!!exception}} defined here with ECMAScript's
+{{ECMAScript/SyntaxError}}. The ECMAScript one should not be used outside
+ECMAScript and it is deliberately omitted from [=simple exception=].
```

Comment:
Cannot use "should not" in a note. If you make it a statement of fact by using "cannot" it's fine.

Reply:
I see.  Does the revised note look okay?  If adding an example "such as~" is not acceptable, it would be okay to omit that part and simply say "The ECMAScript one is delibarately ommitted from simple exception".